pages/index.tsx

```diff
@@ -1,8 +1,21 @@
 import Head from 'next/head';
-import { Inter } from 'next/font/google';
+import { Inter, Montserrat, Manrope } from 'next/font/google';
 import styles from '@/styles/Home.module.css';
 
-const inter = Inter({ subsets: ['latin'] });
+const inter = Inter({
+  subsets: ['latin', 'cyrillic'],
+  weight: ['400', '500', '600'],
```

Comment:
Do we need all of the weights and both subsets? This font seems to be used just once
![Screenshot 2024-05-15 at 10 18 29 AM](https://github.com/fpandyz/best-by-landing/assets/39587602/d7ad1fbc-22b7-430a-b055-2b3c44e62091)